helpers/kubernetes/cluster.go

```diff
@@ -610,7 +611,8 @@ func (c *Cluster) GetVersion() (string, error) {
 
 // ListIngressRoutes returns a list of all routes for ingresses in `namespace` with the given selector
 func (c *Cluster) ListIngressRoutes(ctx context.Context, namespace, name string) ([]string, error) {
-	ingress, err := c.Kubectl.NetworkingV1().Ingresses(namespace).Get(ctx, name, metav1.GetOptions{})
+	ingress, err := c.Kubectl.NetworkingV1().Ingresses(namespace).Get(
+		ctx, names.GenerateResourceName("i-"+name), metav1.GetOptions{})
```

Comment:
I think it should be the caller that worries about passing the right ingress name. This method can stay generic by expecting the correct name to be passed.

Reply:
Done with 89db30b8515d48c9fa4fa9207b7cb352090912ac